website_product_document_download_counter/tests/test_product_document_model.py

```
@@ -0,0 +1,55 @@
+
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase, tagged
+
+from .product_document_test_utils import ProductDocumentTestUtils
+
+
+@tagged("post_install", "-at_install")
+class TestProductDocumentDownloadCounter(TransactionCase, ProductDocumentTestUtils):
+    def setUp(self):
+        super().setUp()
+        self.product, self.attachment, self.document = self._create_product_with_document()
```

Ruff (0.11.9)

Line too long (91 > 88)

(E501)